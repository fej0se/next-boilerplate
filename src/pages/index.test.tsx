import { render } from '@testing-library/react'

import Home from '.'

describe('<Test />', () => {
  it('should render the heading', () => {
    const { container } = render(<Home />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
