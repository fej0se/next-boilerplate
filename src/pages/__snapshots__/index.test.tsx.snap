// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Test /> should render the heading 1`] = `
.c0 {
  background-color: #fefefe;
  color: #000000;
  width: 100%;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  margin: 0;
  line-height: 1.15;
  font-size: 4rem;
}

.c2 {
  padding-top: 2rem;
  margin: 0;
  line-height: 1.15;
  font-size: 2rem;
}

<main
  class="c0"
>
  <h1
    class="c1"
  >
    Next.js boilerplate
  </h1>
  <h2
    class="c2"
  >
    This is a component test.
  </h2>
</main>
`
